README.md

```diff
@@ -75,6 +75,10 @@ To run this:
 
 `$ node ./src/bin/web-server.js`
 
+RPDE APIs are meant to be called directly by users. They are:
```

Comment:
Shall we add a web server endpoints section here, then I can also add mine too

Reply:
Because ours are different things, that's why I did diff sections. So far there is no user story for users accessing your directly, only the status page needs to. So I was thinking a different section with a note "There are meant to be accessed directly by the Status app, and are tailored to it" or something.

Reply:
Basically, cos the status app is currently going to have no local caching the results of this end point will need to be tailored heavily to it and so we want to be able to change in regularly as we need to for the status page. So I don't want users using the same end point cos they will need stability. If a user story does emerge about users wanting to directly access it I'd rather they had diff end points.

But that's almost a side point. I don't really mind about this, it'll might get tweaked later when we put in different system anyway.

Do you want to merge this and rewrite in yours?